src/Core/MySQL/PacketsProtocolBinary.cpp

```diff
@@ -30,17 +25,21 @@ ResultSetRow::ResultSetRow(const Serializations & serializations_, const DataTyp
     : row_num(row_num_), columns(columns_), data_types(data_types_), serializations(serializations_)
 {
     payload_size = 1 + null_bitmap_size;
-    FormatSettings format_settings;
+    static FormatSettings format_settings;
```

Comment:
Please either remove `static` or add a comment that `static` may cause eye cancer :-)